src/units/pathfinder2remaster/player-core/ancestry/elf/heritage/heritage-elf-ancient.yml

```diff
@@ -0,0 +1,15 @@
+unit: heritage/elf/remaster/ancient
+in: heritage/elf
+group: "_{Player Core}"
+name: "_{Ancient Elf}"
+
+inc:
+  - set: heritage
+    value: "_{Ancient Elf}"
+
+  - at: '@heritage'
+    replace:
+      - article:
+        title: "_{Ancient Elf}"
+        contents:
+          - p: "_{Gain a multiclass dedication feat.}"  
```

Comment:
If there's enough space, it would be good to have a place for the bonus feat this grants.
```
- p:
  title: "_{Multiclass dedication}
- article: "##/multiclass-dedication"
  lines: 4
  reduce: 2
```